librocksdb_sys/src/lib.rs

```diff
@@ -71,8 +71,8 @@ pub enum DBEntryType {
     Other = 4,
 }
 
-#[derive(Copy, Clone, Debug, PartialEq)]
-#[repr(C)]
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+#[repr(u8)]
```

Comment:
Personally I think it's wrong.
Our tikv code base relying on sizeof<CompressionType> == sizeof<c_int>